desktop/apps/consign/components/checkbox_input/index.js

```diff
@@ -15,8 +15,8 @@ export default function CheckboxInput (props) {
   const b = block('consignments-submission-checkbox-input')
 
   return (
-    <div className={b('wrapper').mix('artsy-checkbox')} onClick={() => onChange(!value)}>
-      <div className={b('input').mix('artsy-checkbox--checkbox')}>
+    <div className={b.builder()('wrapper').mix('artsy-checkbox')} onClick={() => onChange(!value)}>
```

Comment:
@sweir27 - Just confirming this works without the tailing function-call -- `b.builder()('wrapper').mix('artsy-checkbox')()`?

Reply:
It does seem to work... although it's weird because you're right the documentation says you need the extra `()`. I can add them in but this appears to be OK.

Reply:
Ok as long as it works, np! Will merge